### DISCUSSION TITLE
CMDT where clause

### DISCUSSION THREAD

**Discussion by jasoncoorsh** [jamessimone/apex-rollup#330] [General]
I have a CMDT Rollup with both the parent and child being a custom object. If I leave the Where clause blank the full calc rollup populates the target field correctly. If I enter my desired Where clause the target rollup field is blank for all records. Am I doing something wrong? I'm trying to filter the rollup to sum only Billable work records. Without the Where clause I get both Billable and Non-Billable in the rollup.  
 
![image](https://user-images.githubusercontent.com/107202894/172950120-9cc9c2ff-d7ae-4632-b667-544d7851f384.png)

**Comment by jamessimone** [comment]
@jasoncoorsh there’s nothing visibly wrong with what you’ve entered there — did you copy the api name for `IsBillable__c` exactly? Otherwise — I would enable the `Is Rollup Logging Enabled?` checkbox on the default rollup control CMDT record and keep your apex Developer Console open while running the full recalc. The debug logs that get generated should contain quite a bit of info (check the box for “Debug Only” to filter them down to the relevant bits) and will hopefully show what the issue is. I have many people using checkbox fields in their where clauses without issue, so I’m hopeful that we will be able to easily get to the bottom of this!

**Comment by jasoncoorsh** [comment]
@jamessimone I have enabled logging and checked Debug Only. I don't see anything in the logs that would indicate there is something wrong but I don't know what I'm looking for either.

**Reply by jamessimone** [reply]
@jasoncoorsh is it possible for you to send me one of the logs to look at? My contact info is at the bottom of the Readme. If not, we will figure something else out

**Reply by jamessimone** [reply]
@jasoncoorsh following up here for maximum visibility - I sent you two responses via my work email just now. We've got options! Feel free to respond by email or here - and thanks!

**Reply by jasoncoorsh** [reply]
Thanks for the help @jamessimone . Lowering the Max Lookup Rows Before Batching number to 2000 and the Batch Chunk Size from 2000 to 1000 resolved the issue.
